engine/apps/twilioapp/status_callback.py

```diff
@@ -25,58 +25,63 @@ def update_twilio_call_status(call_sid, call_status):
 
     if call_sid and call_status:
         logger.info(f"twilioapp.update_twilio_call_status: processing sid={call_sid} status={call_status}")
-        status = TwilioCallStatuses.DETERMINANT.get(call_status)
+        status_code = TwilioCallStatuses.DETERMINANT.get(call_status)
+
+        if status_code is None:
+            logger.warning(
```

Comment:
how normal is receiving unexpected status? This code won't alert us if we start receiving unexpected statuses

Reply:
It's ok, status is only needed for better timeline. The thing about checking status always was in twilio part, so I decided to at least log it